### PR TITLE
Welcome email: Add Welcome email text field

### LINKED ISSUE #83084: Welcome email: add textarea to Calypso
Add textarea to customize the welcome message to Calypso.

- Needed: copy

### PR DESCRIPTION
**Pull request by TimBroddin**
Closes https://github.com/Automattic/wp-calypso/issues/83084

## Proposed Changes

This PR adds the textarea that allows you to change the welcome email text.

CleanShot 2023-10-19 at 14 26 36@2x

## Test instructions

1. Apply this PR
2. Navigate to Settings > Newsletter
3. Confirm the field is there & works

The field will be empty, in order to see the default value you need to apply this PR to your sandbox: https://github.com/Automattic/jetpack/pull/33677 

This feature is currently limited to Simple sites, because it needs a Jetpack release for Atomic and self-hosted.

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by phcp — APPROVED** [review]
Nice job @TimBroddin!

**Comment by simison** [comment]
Can the welcome message be first of the three; doesn't need to be in this PR and fine as a follow-up.

**Comment by simison** [comment]
@crisbusquets do you have copy suggestions or these are good to go? Feels little passive but it's settings after all. (Copy changes don't need to block merging these)

I added string freeze label to get these to translations before merging already.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9423975

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @TimBroddin for including a screenshot in the description! This is really helpful for our translators.

**Comment by TimBroddin** [comment]
> @crisbusquets do you have copy suggestions or these are good to go? Feels little passive but it's settings after all. (Copy changes don't need to block merging these)
> 
> I added string freeze label to get these to translations before merging already.

I'll wait to merge until the translations are in.

**Comment by crisbusquets** [comment]
> @crisbusquets do you have copy suggestions or these are good to go? Feels little passive but it's settings after all. (Copy changes don't need to block merging these)

I think they're good for a first iteration, they're easy to understand.

Will we have tracking here? For example, to know how many people customize the welcome message. Maybe it's a tracking we don't need now, but who knows in the future.

**Comment by simison** [comment]
Is the default value empty, and when empty, the email isn't sent out?

**Comment by TimBroddin** [comment]
> Is the default value empty, and when empty, the email isn't sent out?

The default value is what used to be there: "Cool, you are now subscribed to %1$s and will receive an email notification when a new post is published."

> Will we have tracking here? For example, to know how many people customize the welcome message. Maybe it's a tracking we don't need now, but who knows in the future.

There's a tracks event that fires, when any of the two fields change, named `calypso_settings_newsletter_saved`, but it doesn't include which field actually changes.

We do show changes in the activity log, so it's definitely stored somewhere, I just can't surface it 😀

image

**Comment by simison** [comment]
> "Cool, you are now subscribed to %1$s and will receive an email notification when a new post is published."

👍 We'll need to work on that copy but later. :-)

Screenshot for the record 

image

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9423975).

**Pull request merged**
